packages/reports/src/Components/Errors/error-component.tsx

```diff
@@ -1,23 +1,34 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { PageError, Dialog } from '@deriv/components';
 import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 
+type TErrorComponent = {
+    header?: string;
+    is_dialog?: boolean;
+    message?: React.ReactElement | string;
+    redirect_label?: string;
+    redirectOnClick?: () => void;
+    should_show_refresh?: boolean;
+    type?: string;
+};
+
 const ErrorComponent = ({
     header,
     message,
     is_dialog,
     redirect_label,
     redirectOnClick,
     should_show_refresh = true,
-}) => {
+}: TErrorComponent) => {
     const refresh_message = should_show_refresh ? localize('Please refresh this page to continue.') : '';
 
     return is_dialog ? (
         <Dialog
             title={header || localize('There was an error')}
+            dismissable={false}
```

Comment:
Is there a reason you're adding these props here? Should they have default values?

Reply:
It does not have a default value.